Mathlib/RingTheory/Ideal/Operations.lean

```diff
@@ -823,9 +824,15 @@ instance {R : Type*} [CommSemiring R] [NoZeroDivisors R] : NoZeroDivisors (Ideal
   eq_zero_or_eq_zero_of_mul_eq_zero := mul_eq_bot.1
 
 /-- A product of ideals in an integral domain is zero if and only if one of the terms is zero. -/
+@[simp]
+lemma multiset_prod_eq_bot {R : Type*} [CommRing R] [IsDomain R] {s : Multiset (Ideal R)} :
+    s.prod = ⊥ ↔ ⊥ ∈ s :=
+  Multiset.prod_eq_zero_iff
+
+/-- A product of ideals in an integral domain is zero if and only if one of the terms is zero. -/
+@[deprecated multiset_prod_eq_bot]
```

Comment:
```suggestion
@[deprecated multiset_prod_eq_bot] -- since 26 Dec 2023
```

Reply:
I think we could just use [git blame](https://github.com/leanprover-community/mathlib4/blame/45cc08d2e8f24d3a80ecb750023ab5f498d5ca2c/Mathlib/Algebra/GroupWithZero/NonZeroDivisors.lean#L237) to keep track of the date:
![image](https://github.com/leanprover-community/mathlib4/assets/3064145/062f4c25-a359-4956-8d9b-63bef383eb8f)